xds/internal/server/listener_wrapper.go

```diff
@@ -297,11 +301,22 @@ func (l *listenerWrapper) Accept() (net.Conn, error) {
 			conn.Close()
 			continue
 		}
-		// TODO: once matched an accepted connection to a filter chain,
-		// instantiate the HTTP filters in the filter chain + the filter
-		// overrides, pipe filters and route into connection, which will
-		// eventually be passed to xdsUnary/Stream interceptors.
-		return &connWrapper{Conn: conn, filterChain: fc, parent: l}, nil
+		var rc xdsclient.RouteConfigUpdate
+		if fc.InlineRouteConfig != nil {
+			rc = *fc.InlineRouteConfig
+		} else {
+			rcPtr := l.rdsUpdates
+			rcuPtr := (*map[string]xdsclient.RouteConfigUpdate)(rcPtr)
+			rcu := *rcuPtr
+			rc = rcu[fc.RouteConfigName]
+		}
+
+		if err := fc.ConstructUsableRouteConfiguration(rc); err != nil {
+			l.logger.Warningf("error constructing usable route configuration: %v", err)
```

Comment:
Nit: this is going to be really long error message. For example:
`error constructing usable route configuration: error constructing virtual host: error constructing matcher: <actual error>`.

Reply:
Changed from error constructing x: %v to x construction: %v. Is this what you were going for or is this still too verbose?